src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -102,35 +110,35 @@ bool DebugKeepNode
 
 
 public partial record ProxyForward
-(
-    [PartitionKey] string Region,
+(
+    [PartitionKey] string Region,
```

Comment:
```suggestion
    [PartitionKey] Region Region,
```

There's a few other places in this file where we could be using the `Region` type alias